pkg/progress/progress.go

```diff
@@ -46,12 +52,22 @@ type progressIndicator struct {
 	remaining float64
 	// We use a fixed interval's history to calculate the latest average speed.
 	history *list.List
-	// We use speedStatisticalWindow / updateInterval to get the windowLengthLimit.
-	// Assume that the windowLengthLimit is 3, the init value is 1. after update 3 times with 2, 3, 4 separately. The window will become [1, 2, 3, 4].
+	// We use (speedStatisticalWindowCapacity / updateInterval + 1) to get the windowCapacity.
```

Comment:
Why +1?

Reply:
I write wrong, should be `(speedStatisticalWindowCapacity/updateInterval) + 1`. If the distance of the road is 100KM, and every 1KM needs to set a road sign, then in fact, 101 road signs are needed (0,1,2,3,4... ,99,100). In fact, in the previous implementation, since `PushBack` is after `if p.history.Len() > p.windowLengthLimit`, so the actual capacity is also `speedStatisticalWindow/updateInterval + 1`.